contrib/win32/win32compat/shell-host.c

```diff
@@ -663,37 +747,28 @@ ProcessEvent(void *p)
 		wX = LOWORD(idObject);
 		wY = HIWORD(idObject);
 		
-		SMALL_RECT readRect;
 		readRect.Top = wY;
 		readRect.Bottom = wY;
 		readRect.Left = wX;
-		readRect.Right = ConSRWidth();
```

Comment:
this code is introduced to fix refreshing the client side view for the below example...
User has the executed the below commands in series,
1) x
2) abcd

now user press up arrow.. it will show "abcd".. one more up arrow it should show "x" but if you remove this code it will show "xbcd"..